app/lib/logic/rolling_updates.rb

```diff
@@ -287,6 +287,12 @@ def missing_config
         end
       end
 
+      class PolicyRegistryUi < PolicyRegistry
+        def enabled?
+          super && provider_can_use?(:policy_registry)
```

Comment:
3-level rolling update feature. I think I opened Pandora's box :)

Reply:
Right 🤣 I was thinking the same but I do not see any other solution where you say you need `:policy` and  `:policy_registry`